server/src/main/java/io/spine/server/aggregate/AggregateRepository.java

```diff
@@ -623,10 +631,12 @@ protected A play(I id, AggregateHistory history) {
         tx.commitIfActive();
         if (!success) {
             lifecycleOf(id).onCorruptedState(outcome);
-            throw newIllegalStateException("Aggregate `%s` (ID: %s) cannot be loaded.%n",
+            String outcomeDetails = Json.toJson(outcome);
+            throw newIllegalStateException("Aggregate `%s` (ID: %s) cannot be loaded.%n%s",
```

Comment:
Can we have something explaining the trailing JSON, please?

Reply:
@alexander-yevsyukov 
That's because this JSON will start with `DispatchOutcome`. I thought it was going to be enough.
I will add some "label" before it.